app/views/solutions/show.html.haml

```diff
@@ -44,7 +43,7 @@
     .info-bar
       .lo-container
         This is your published solution.
-        =link_to "Switch to solving mode.", [:my, @solution]
+        %li=link_to "Switch to solving mode.", [:my, @solution]
```

Comment:
Is this needed? It doesn't seem to be within the breadcrumb. Or maybe I just got confused with the spacing.

Reply:
I must've added that by accident. I removed that from the commit.

Reply:
That was really quick :)